AlphaWallet/Transfer/ViewControllers/SendViewController.swift

```diff
@@ -283,8 +283,14 @@ class SendViewController: UIViewController, CanScanQRCode {
         let input = targetAddressTextField.value.trimmed
         targetAddressTextField.errorState = .none
         amountTextField.errorState = .none
-
-        guard let value = viewModel.validatedAmount(value: amountTextField.ethCost) else {
+        let checkIfGreaterThenZero: Bool
+        // allow users to input zero on native transactions as they may want to send custom data
+        if case .nativeCryptocurrency = transferType {
```

Comment:
Can you replace the `if-case` with an explicit `switch-case`? Otherwise it always comes back to bite us.